apps/explorer/src/components/module/module-functions-interaction/ModuleFunction.tsx

```diff
@@ -93,9 +93,9 @@ export function ModuleFunction({
                 transaction: tx,
                 options: {
                     contentOptions: {
-                        showEffect: true,
+                        showEffects: true,
                         showEvents: true,
-                        showInputs: true,
+                        showInput: true,
```

Comment:
previously this bug was uncaught by the compiler because of a wrong type definition in wallet-kit-core/src/index.ts